v23.1/changefeed-sinks.md

```diff
@@ -201,7 +201,7 @@ URI Parameter      | Description
 When using Pub/Sub as your downstream sink, consider the following:
 
 - Pub/Sub sinks support `JSON` message format. You can use the [`format=csv`](create-changefeed.html#format) option in combination with [`initial_scan='only'`](create-changefeed.html#initial-scan) for CSV-formatted messages.
-- You must specify the `region` parameter in the URI to maintain [ordering guarantees](changefeed-messages.html#ordering-guarantees). Unordered messages are not supported, see [Known Limitations](change-data-capture-overview.html#known-limitations) for more information.
+- Use the [`unordered`](create-changefeed.html#unordered) option for multi-region Pub/Sub.
```

Comment:
Maybe a quick explanation of the tradeoff here? Something like "Google cloud's multi-region Pub/Sub will have lower latency when emitting from multiple regions, but Google does not support message ordering for multi-region topics."

Reply:
Added!